yt/frontends/ramses/data_structures.py

```diff
@@ -530,7 +530,7 @@ def read_rhs(f, cast):
                     dom, mi, ma = f.readline().split()
                     self.hilbert_indices[int(dom)] = (float(mi), float(ma))
 
-        if rheader['ordering type'] != 'hilbert' and self.bbox:
+        if rheader['ordering type'] != 'hilbert' and hasattr(self, 'bbox'):
```

Comment:
The attribute is named `_bbox` and is always defined for a RAMSES dataset. I'd suggest the following
```suggestion
        if rheader['ordering type'] != 'hilbert' and self._bbox is not None:
```